src/tests/ftest/recovery/pool_membership.yaml

```
@@ -12,12 +12,29 @@ server_config:
       log_file: daos_server_0.log
       pinned_numa_node: 0
       fabric_iface: ib0
+      targets: 8
     1:
       storage: auto
       fabric_iface_port: 31516
       log_file: daos_server_1.log
       pinned_numa_node: 1
       fabric_iface: ib1
+      targets: 8
 
 pool:
   size: 100G
+
+container:
+  control_method: daos
+  type: POSIX
+
+ior:
+  client_processes:
+    ppn: 1
+  iorflags:
+    flags: -k -D 10 -v -w -W
+    api: DFS
+    transfer_size: 1M
+    block_size: 10G
+    dfs_oclass: SX
+    dfs_dir_oclass: SX
```

No need for `iorflags`
```suggestion
  flags: -k -D 10 -v -w -W
  api: DFS
  transfer_size: 1M
  block_size: 10G
  dfs_oclass: SX
  dfs_dir_oclass: SX
```

---

Done. Thanks.